### PR TITLE
refactor(nwaku)_: store queries

### PR DESCRIPTION
**Pull request by richard-ramos**
Execute store queries with nwaku instead of go-waku
(Still pending replace peerID by peerAddr in the function calls. This is to be done on a separate PR)

### CODE REVIEW

**Review by gabrielmer — APPROVED** [review]
Thank you!

File: vendor/github.com/waku-org/go-waku/waku/v2/api/history/cycle.go [hunk 1]
```diff
@@ -426,14 +406,30 @@ func (m *StorenodeCycle) WaitForAvailableStoreNode(ctx context.Context, timeout
 			select {
 			case <-m.StorenodeAvailableOneshotEmitter.Subscribe():
 			case <-ctx.Done():
+				if errors.Is(ctx.Err(), context.Canceled) {
+					return
```
Comment (Ivansete-status) [inline vendor/github.com/waku-org/go-waku/waku/v2/api/history/cycle.go:410]: I wonder if we should log this condition

**Review by Ivansete-status — APPROVED** [review]
LGTM! Thanks! Just added one naïve comment:)

File: vendor/github.com/waku-org/go-waku/waku/v2/api/history/cycle.go (hunk 1 above)
Comment (richard-ramos) [reply vendor/github.com/waku-org/go-waku/waku/v2/api/history/cycle.go:410]: Shouldn't be necessary, as a cancel would happen when you logout of the app.

**Pull request merged**